src/ducks/modules/stage.js

```diff
@@ -1,13 +1,18 @@
+import { actionTypes as protocolActionTypes } from '../../ducks/modules/protocol';
+
 const NEXT_PROMPT = 'NEXT_PROMPT';
 const PREVIOUS_PROMPT = 'PREVIOUS_PROMPT';
+const NEXT_STAGE = 'NEXT_STAGE';
+const PREVIOUS_STAGE = 'PREVIOUS_STAGE';
 
 const initialState = {
   promptIndex: 0,
```

Comment:
I think this would be best as something like `sessionProgress`, or `sessionManager`. This seems to approximate what was in `session.js` in the old code. 

Great catch on why it needs to be where it is. Hopefully we can come up with something graceful. Let me know when you are finalized on the naming, and we can merge.